README.md

```diff
@@ -4,27 +4,51 @@
 [![bitHound Dependencies](https://www.bithound.io/github/onfido/onfido-sdk-ui/badges/dependencies.svg)](https://www.bithound.io/github/onfido/onfido-sdk-ui/master/dependencies/npm)
 [![Build Status](https://travis-ci.org/onfido/onfido-sdk-ui.svg?branch=master)](https://travis-ci.org/onfido/onfido-sdk-ui)
 
-## Overview
+## Table of contents
+
+* [Overview](#overview)
+* [Getting started](#getting-started)
+* [JSON Web Token authentication](#json-web-token-authentication)
+* [Handling callbacks](#handling-callbacks)
+* [Customising SDK](#customising-sdk)
+* [Uploading files to Onfido API](#uploading-files-to-onfido-api)
+* [Creating checks](#creating-checks)
+* [Going live](#going-live)
 
-This is a plug-and-play SDK that leverages the Onfido SDK core, helping users take document and face captures that can then be sent to our backend APIs.
+## Overview
 
-All document captures are sent over WebSockets to our image evaluation API, to ensure your users are submitting a document image of adequate quality.
+This SDK provides a set of [React](https://facebook.github.io/react/) components for JavaScript applications to allow capturing of identity documents and face photos for the purpose of identity verification. The SDK offers a number of benefits to help you create the best onboarding / identity verification experience for your customers:
```

Comment:
React is an implementation detail - I don't think it's relevant to someone integrating with the SDK